projects/bp-gallery/src/components/views/search/helpers/getSearchResultPictureIds.ts

```diff
@@ -0,0 +1,60 @@
+import { MeiliSearch } from 'meilisearch';
+
+const dateToTimeStamp = (date: string) => {
+  return Date.parse(date) / 1000;
+};
+
+const getSearchResultPictureIds = async (
+  searchparams: {
+    searchTerms: string[];
+    searchTimes: string[][];
+  },
```

Comment:
You can use destructuring here to avoid the two lines below (i. e. `{ searchTerms, searchTimes }: { searchTerms: string[]; searchTimes: string[][]; }`)